turbinia/api/cli/turbinia_client/core/commands.py

```diff
@@ -33,7 +33,7 @@
 from turbinia_client.core import groups
 from turbinia_client.helpers import formatter
 
-log = logging.getLogger('turbinia')
+log = logging.getLogger(__name__)
```

Comment:
Why change the logger name here?  Not sure what all this affects, but that might be why you had to explicitly set some logger settings in some modules if it's not using the base `turbinia` logger.

Reply:
The CLI tool has it's own logger in ```turbiniactli_tool.py``` since it's packaged separately from the main Turbinia package. The name here is not super important and doesn't relate to the 'turbinia' logger used in other modules.